packages/tsx/tsx.mjs

```diff
@@ -5,12 +5,24 @@ import { getFilenameExt } from '@nodejs-loaders/parse-filename';
 import { findEsbuildConfig } from './find-esbuild-config.mjs';
 
 
+/**
+ * The load hook needs to know the parent URL to find the esbuild config.
+ * But the resolve hook doesn't have access to the parent URL.
+ * If you try to pass it as return value from the resolve hook, it will be overwritten by node
+ */
+let parentURL = null;
```

Comment:
I think this will not work: `parentURL` is different per module, so this will constantly get overwritten making the value always wrong. I think you'll need to make a Map keyed by `nextResult.url`:

```suggestion
 * @type {Map<URL['href'], URL['href']>}
 */
const parentURLs = new Map();
```

Reply:
intressing